owasp-top10-2017-apps/a3/snake-pro/app/api/routes.go

```diff
@@ -55,6 +56,9 @@ func Register(c echo.Context) error {
 	userData.UserID = newGUID1.String()
 	userData.HighestScore = 0
 
+	password, err := bcrypt.GenerateFromPassword([]byte(userData.Password), 14)
```

Comment:
@danilogila could you please check if `bcrypt.GenerateFromPassword([]byte(userData.Password), 14)` returns an error?

Reply:
It didn't return an error. I added an return statement at my last PR.

Reply:
Although it did not return an error, for security reasons, it always good to add instructions if this error occurs (`!= nil`)